tests/test_dcrModel.py

```diff
@@ -341,9 +424,9 @@ def testRotationFlipped(self):
                                         crval=visitInfo.getBoresightRaDec(),
                                         flipX=flip)
                 rotAngle = calculateImageParallacticAngle(visitInfo, wcs)
-                if flip:
+                if ~flip:
```

Comment:
This confuses me.  Do you mean `not flip`? I think that `flip` is always a Boolean, in which case this `if` statement seems to always be true (`~True == -2` and `~False == -1`), which now makes me doubt the test.

Reply:
Thanks for catching this! That was a dumb mistake: it should just be `if flip:`.